scripts/conda-environment.yml

```diff
@@ -17,3 +17,4 @@ dependencies:
     - py_trees==0.7.6
     - sysv-ipc
     - tk
+    - pynput
```

Comment:
this is available from conda-forge, so move it up there. We only use `pip` when something is not available in conda-forge.